README.md

```diff
@@ -82,23 +78,41 @@ npm install
 
 - Looking for a component? [JS.coach](https://js.coach/react-native)
 - Fellow developers write and publish React Native modules to npm and open source them on GitHub.
-- Many third party modules may be missing Windows support, so reach out to the project owners directly.
+- Third party modules may be missing Windows support, so reach out to the project owners directly.
 - Making modules helps grow the React Native ecosystem and community. We recommend writing modules for your use cases and sharing them on npm.
+- You should almost never need to modify React Native Windows itself to create custom controls
```

Comment:
Maybe clarify here what you mean by "React Native Windows"  (e.g. The ReactNative project in the SLN file that was created) or something